README.md

```diff
@@ -1,9 +1,11 @@
 [![Juju logo](doc/juju-logo.png?raw=true)](https://juju.is/)
 
-[Juju is a model-driven **Operator Lifecycle Manager**
-(OLM)](https://juju.is/overview). Juju greatly improves the experience of
-running Kubernetes operators, especially in projects that integrate many
-operators from different publishers.
+Juju is an Open Source Charmed Operator Framework. It helps you move from configuration management to application management and has two main components:
+
+* **Charmed Operator Lifecycle Manager (OLM)** - a hybrid-cloud application management and orchestration system that helps you from Day 0 to Day 2. Deploy, configure, scale, **integrate**, maintain and manage Kubernetes native, container-native and VM-native applications -- and the relations between them.
+   * **Charmed Operators, packaged as “Charms”**, are software that encapsulate a single application and all the code and know-how it takes to operate it on Kubernetes or machines.
```

Comment:
This looks a bit wrong with regards to indentation, and indeed doesn't quite render correctly

Reply:
It renders well for me - I mean, it renders as _I_ was expecting. 
![image](https://user-images.githubusercontent.com/5586487/115519889-86253b80-a281-11eb-946a-e5495a690320.png)

Reply:
The middle line looks out of place - I guess you're saying the extra indentation is intended?
Wouldn't it look better to have all 3 bullet points aligned?

Reply:
It would, but the second item is a sub-section of the first one.

Reply:
It seems clumsy. How about this instead (we define for the user what a charm is up front and then list the components of the framework. It avoids the need to explain what a charm is inside the list and IMO is a better way to do it).

Juju is an Open Source Charmed Operator Framework. It helps you move from configuration management to application management.

**Charmed Operators, packaged as “Charms”**, are software that encapsulate a single application and all the code and know-how it takes to operate it on Kubernetes or machines.

The Juju Cjarmed Operator Framework has two main components:

* **Charmed Operator Lifecycle Manager (OLM)** - a hybrid-cloud application management and orchestration system that helps you from Day 0 to Day 2. Deploy, configure, scale, **integrate**, maintain and manage Kubernetes native, container-native and VM-native applications -- and the relations between them.
* **Charmed Operator SDK** - a guide to help you build Charmed Operators for use with the Charmed OLM.